tests/unit/fair_queue_test.cc

```diff
@@ -55,15 +55,15 @@ struct request {
 };
 
 class test_env {
-    fair_group _fg;
+    shared_throttle _fg;
     fair_queue _fq;
     std::vector<int> _results;
     std::vector<std::vector<std::exception_ptr>> _exceptions;
     fair_queue::class_id _nr_classes = 0;
     std::vector<request> _inflight;
 
-    static fair_group::config fg_config(unsigned cap) {
-        fair_group::config cfg;
+    static shared_throttle::config fg_config(unsigned cap) {
```

Comment:
Minor: maybe we could rename also the function to `throttle_config()`?

Reply:
Okay, please ignore the comment -- I see that at the end of patches we remove it completely.